src/DacpacTool/ScriptParser.cs

```diff
@@ -24,8 +25,20 @@ private void Parse()
         {
             if (! _parsed)
             {
-                _parser.Parse();
-                _parsed = true;
+                try
+                {
+                    _parser.Parse();
+                    _parsed = true;
+                }
+                catch (Microsoft.SqlTools.ServiceLayer.BatchParser.BatchParserException ex)
+                {
+                    string mostRecentFile = _includedFileNames.LastOrDefault();
+                    if (mostRecentFile != null)
```

Comment:
I would prefer not to have two different throw statements, but put the exception message into a variable (possibly using a ternary operator) and then a single throw statement.

Reply:
heh, that was my original approach but I decided I didn't like it. 😄 I don't have a strong opinion, though, just thought it looked a little messy with the ternary. I'll go back to this approach with the next commit.